packages/oauth2/src/access-token/parse-access-token-request.ts

```diff
@@ -60,15 +58,15 @@ export interface ParseAccessTokenRequestOptions {
  * that can be returned to the client.
  */
 export function parseAccessTokenRequest(options: ParseAccessTokenRequestOptions): ParseAccessTokenRequestResult {
-  const parsedAccessTokenRequest = v.safeParse(vAccessTokenRequest, options.accessTokenRequest)
+  const parsedAccessTokenRequest = vAccessTokenRequest.safeParse(options.accessTokenRequest)
   if (!parsedAccessTokenRequest.success) {
     throw new Oauth2ServerErrorResponseError({
       error: Oauth2ErrorCodes.InvalidRequest,
-      error_description: `Error occured during validation of authorization request.\n${JSON.stringify(valibotRecursiveFlattenIssues(parsedAccessTokenRequest.issues), null, 2)}`,
+      error_description: `Error occured during validation of authorization request.\n${JSON.stringify(parsedAccessTokenRequest.error.issues, null, 2)}`,
```

Comment:
Might be nice to still keep this method and just return the input for now. That'll make it easy to add whatever logic we want to when we want to revisit this

so something like `zodErrorToHumanReadable`

Reply:
I see there are several utils availalbe in zod on an error:
- `.format`
- `.flatten`

`.format` seems to give me a good result